src/main/resources/vipconfig.properties

```diff
@@ -22,4 +22,5 @@ collectSource = false
 cleanCache = false
 cacheExpiredTime = 8600000
 machineTranslation = false
-defaultLocale = en
\ No newline at end of file
+defaultLocale = en
+prodMode = true
```

Comment:
This should be false by default.

Reply:
It is false by default in the code. This is just for testing purposes.